create_box/main.py

```diff
@@ -1,24 +1,19 @@
+"""
+Here is where I read the code from
+https://stackoverflow.com/questions/32954820/how-to-make-a-rectangle-consisting-of-the-same-character
+"""
 def create_box(height, width, character):
```

Comment:
box = ""

    for row in range(height):
       box += character * width + "\n"

    return box